### DISCUSSION TITLE
导入CSV数据问题，分隔符“,”，但是有些取值字符串里面含有"," 请问该怎么处理

### DISCUSSION THREAD

**Discussion by yorkexing** [apache/doris#16851] [A - General / Q&A]
如：
0,,,,"100|0,-1|1,62|2,271|3,677|4,1286|5,2070|6,2978|7,3927|8,4826|9,5630|10,6325|11,6923|12,7441|13,7883|14,8256|15,8574|16,8853|17,9115|18,9346|19,9541|20,9712|21,9848|"

"100|0,-1|1,62|2,271|3,677|4,1286|5,2070|6,2978|7,3927|8,4826|9,5630|10,6325|11,6923|12,7441|13,7883|14,8256|15,8574|16,8853|17,9115|18,9346|19,9541|20,9712|21,9848|" 这个是一个字段

**Comment by zy-kkk** [comment]
If you are using a csv file to stream load locally, this situation cannot be handled. If you are using spark/flink to write data, you can specify special delimiters such as &,$, or import in json format

**Comment by yorkexing** [comment]
我是放入hdfs，通过broker导入
目前的解决办法是，把数据集重新处理了，加了特殊符号分割
但是希望能够支持CSV的解析，比如我现在用pandas read_csv是能够正确读取列的，而不受数据里面的 逗号 影响，因为任何的特殊字符都有可能出现在数据cell里面

**Reply by zy-kkk** [reply]
If you need a new requirement, write down an issue that describes the problem you are currently facing and how you would like to solve it
